data/vocab.json

```diff
@@ -2121,6 +2121,15 @@
                 }
             ]
         },
+	"unaltered function": {
```

Comment:
Is this description copied from civic? If so could you set the source for this record to "civic" (ex."source": "civic")? Just for the record itself, not the links

Reply:
I didn't get this description from civic, so I will leave it as is